app/views/admin/sponsors/_form.html.haml

```diff
@@ -1,66 +1,68 @@
 = simple_form_for [:admin, @sponsor], :html => {:multipart => true, novalidate: true } do |f|
   .row
-    .large-6.columns
+    .col-12.col-md-6
       .row
-        .large-12.columns
+        .col
           = f.input :name
 
       .row
-        .large-12.columns
+        .col
           = f.input :website, as: :url
 
       .row
-        .large-12.columns
+        .col
           = f.input :description
 
       - if current_user.has_role?(:admin)
         .row
-          .large-6.columns
+          .col-6
             = f.input :level, collection: Sponsor.levels.keys, label_method: :humanize
 
-      .row
-        .large-12.columns.admin_margin
+      .row.mt-3
+        .col
           = f.input :avatar, as: :file, required: !@sponsor.avatar?
           = f.hidden_field :image_cache
 
       .row
-        .large-12.columns
-          .panel.callout#contacts
-            %strong Contact Details
-            %p Required so that all organisers can get in touch with sponsor when required. Please do not use your personal or chapter details.
-            = f.simple_fields_for :contacts do |c|
-              = render 'contact_fields', f: c
-            .links
-              = link_to_add_association 'Add contact', f, :contacts, class: 'button tiny'
+        .col
+          #contacts.card.bg-light.border-info.mb-4.mb-md-0
+            .card-body
+              %strong Contact Details
+              %p Required so that all organisers can get in touch with sponsor when required. Please do not use your personal or chapter details.
+              = f.simple_fields_for :contacts do |c|
+                = render 'contact_fields', f: c
+              .links
```

Comment:
@KimberleyCook is this a Bootstrap class? If not can we replace it with a Bootstrap one?

Reply:
This is needed. When removed it breaks a bunch of UI when "Add contact" is pressed.